eth_typing/abi.py

```diff
@@ -54,17 +55,17 @@ class ABIEventParam(TypedDict, total=False):
     """Type of the event parameter."""
 
 
-class ABIComponent(TypedDict):
+class ABIComponent(TypedDict, total=False):
```

Comment:
We should remove this as well though, right?

Reply:
Dang that snuck in, fixing!